setup.py

```diff
@@ -14,6 +15,27 @@
 
 from setuptools import find_packages
 from setuptools import setup
+from setuptools.command.test import test as TestCommand
+
+
+class PyTest(TestCommand):
```

Comment:
`.travis.yml` doesn't contain anything in regards to `setup.py`. Only `tox.ini`:

```ini
commands =
    coverage run setup.py test
```

Could be easily replaced with these lines (tested, works on my computer):

```ini
commands =
  coverage run -m pytest -v
```

I think, that can be replaced without breaking too much things. :grinning:

Not sure if we can further and replace coverage with `pytest-cov`:

```ini
commands =
   py.test --no-cov-on-fail --cov=dbxincluder --cov-report=term-missing [--cov-fail-under=X]
```